pages/en/issuance-email.md

```diff
@@ -31,9 +31,14 @@ to best solve this problem.)
 
 The validity period of this e-mail address attribute is *one year*.
 
-Immediately after issuance, the foundation removes this e-mail
-attribute from its own systems. The foundation does not keep a log of
-issuance.
+**Note:**
+As of November 30th, 2020, the issuance of e-mail credentials is no longer
```

Comment:
I though in English its `Email` (so without a dash)

Reply:
Both are fine actually, I used e-mail with hyphen for consistency with the rest of the page.